libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -1488,3 +1488,9 @@ class NavEKF3_core : public NavEKF_core_common
     bool EKFGSF_run_filterbank;             // true when the filter bank is active
     uint8_t EKFGSF_yaw_valid_count;         // number of updates since the last invalid yaw estimate
 };
+
+// accessor macros to make the code more readable and less error prone
+#define VarField(field) (offsetof(state_elements,field)/sizeof(ftype))
```

Comment:
can't we have an inline function instead ? 
it just that macro tend to bother debuging with GDB ...